pkg/artifact/ingest_logs.go

```diff
@@ -88,12 +95,21 @@ func NewLogIngester(ctx context.Context, logsBucketName, gitHubAppID, gitHubInst
 		return nil, fmt.Errorf("failed to get github app installation: %w", err)
 	}
 
+	ghClient, err := githubclient.NewWithPermissions(ctx, gitHubAppID, gitHubInstallID, gitHubPrivateKey, map[string]string{
```

Comment:
Were mixing the existing impl that uses the token from an installation with generating a github client.

I think we should create a new github client in this func and use it in the existing `handleMessage` func. Instead of creating a raw http request we should use `client.NewRequest` to use the token from the github client.

The permissions should be pull_request:write and actions:read for all repos.

@bradegler  thoughts?

Reply:
Agreed. We should not mix and match.

Reply:
Refactored `handleMessage` to use the client, PTAL.